File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLCandlesticksGraphProps.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLCandlesticksGraphProps
    {
        public String CanvasID;
        public String WindowID;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public List<Object> Data;
        public List<Object> XMarksLabelData;
        public String XMarksWidth;
        public String YMaxValue;
        public String NumMarksY;
        public String Title;
        public String TitleColor;
        public String TitleHeight;
        public String TitleFontString;
        public String CandleBodyWidth;
        public String CandleBodyColor;
        public String CandleLineColor;
        public String MarginLeft;
        public String AxisLabelsColor;
        public String AxisLabelsHeight;
        public String AxisLabelsFontString;
        public Object Tag;
        public String TabStopIndex;

        CCLCandlesticksGraphProps()
        {
            Data = new ArrayList<Object>();
            XMarksLabelData = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLCandlesticksGraphProps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLCandlesticksGraphProps
    {
        public class XMarkLabel
        {
            public int Value { get; set; }
            public string Label { get; set; }
        }

        public string CanvasID { get; set; }
        public int WindowID { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public List<List<int>> Data { get; set; }
        public List<XMarkLabel> XMarksLabelData { get; set; }
        public int XMarksWidth { get; set; }
        public int YMaxValue { get; set; }
        public int NumMarksY { get; set; }
        public string Title { get; set; }
        public string TitleColor { get; set; }
        public int TitleHeight { get; set; }
        public string TitleFontString { get; set; }
        public string CandleBodyWidth { get; set; }
        public string CandleBodyColor { get; set; }
        public string CandleLineColor { get; set; }
        public int MarginLeft { get; set; }
        public string AxisLabelsColor { get; set; }
        public int AxisLabelsHeight { get; set; }
        public string AxisLabelsFontString { get; set; }
        public object Tag { get; set; }
        public int TabStopIndex { get; set; }

        public CCLCandlesticksGraphProps()
        {
            Data = new List<List<int>>();
            XMarksLabelData = new List<XMarkLabel>();
        }
    }
}